evap/results/tools.py

```diff
@@ -164,7 +164,9 @@ def _calculate_results_impl(course):
             elif question.is_heading_question:
                 results.append(HeadingResult(question=question))
 
-        section_warning = questionnaire_max_answers[(questionnaire, contribution)] < questionnaire_warning_thresholds[questionnaire]
+        section_warning = False
+        if any(question.is_rating_question for question in questionnaire.question_set.all()):
```

Comment:
i think it would make more sense to check whether there is any rating question in `results`, because that's what's shown in the UI and not `questionnaire.question_set.all()`.

Reply:
check